maintainers/maintainer-list.nix

```diff
@@ -9469,6 +9469,12 @@
     githubId = 378734;
     name = "TG ⊗ Θ";
   };
+  tgunnoe = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add tgunnoe``.

Reply:
Done, thanks!